mlflow/deployments/cli.py

```diff
@@ -0,0 +1,154 @@
+import click
```

Comment:
We should also add the `predict` function to the CLI + tests, but I think that can come in a follow-up to help expedite merging this PR first :)